BitwardenShared/Core/Auth/Services/KeychainRepository.swift

```diff
@@ -42,6 +42,21 @@ enum KeychainItem: Equatable {
         }
     }
 
+    /// The protection level for this keychain item.
+    var protection: CFTypeRef {
+        switch self {
+        case .biometrics,
+             .deviceKey,
+             .neverLock,
+             .pendingAdminLoginRequest,
+             .refreshToken:
+            kSecAttrAccessibleWhenUnlockedThisDeviceOnly
+        case .accessToken,
+             .authenticatorVaultKey:
+            kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly
```

Comment:
❓ Does it make sense to also do this for the `refreshToken`? If the access token is expired, we'll need to use the refresh token to fetch a new access token before proceeding.

Reply:
Sounds good. I was trying to keep as few things as possible from moving to the wider protection setting, but both you and Fede mentioned this and I think it makes sense to move refreshToken there as well. I'll put up a commit for that. 👍